src/js/ui/header/index.js

```diff
@@ -50,9 +44,13 @@ pageTitle.innerText = 'Standard';
  *
  */
 export const checkState = () => {
```

Comment:
The name checkState does not really imply what this function is doing. What could be a better name here?

Reply:
Also, how about we use this function to render the button directly in the header, instead of exporting it and running it in `./src/js/ui/index.js`. We could simply do something like this:

export const header = () => {
...
return (
  // ...rest of code
  <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
    ${checkState()}
  </div>
  // ...
)
}